src/plugin_host.jl

```diff
@@ -119,7 +120,7 @@ macro fnsync(args...)
 end
 
 function fun(ex)
-    if ex.head == :block && length(ex.args) == 2 && ex.args[1].head == :line
+    if ex.head == :block && length(ex.args) == 2 # && ex.args[2].head == :line
```

Comment:
Remove comment

Reply:
Fixed in https://github.com/bfredl/Neovim.jl/commit/97ae01b61b6696dcbf3ed8ccfd285658311cb256